knowledge-base/editor-add-class-to-selected-text.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
The suggested approach has no flexibility. For example, if you select one work from a sentence, the Editor will split it to a new paragraph. If we want to have such a KB, the custom algorithm should support:

* Block and inline content.
* Different HTML tags.